examples/regexp-type/regexp_type.bal

```diff
@@ -3,10 +3,16 @@ import ballerina/lang.regexp;
 
 public function main() returns error? {
     
-    // Declare a RegExp using regular expression literal.
-    string:RegExp _ = re `[a-zA-Z0-9]`;
+    // Declare a `RegExp` value using a regular expression literal.
+    regexp:RegExp alphanumericPattern = re `[a-zA-Z0-9]`;
```

Comment:
Shall we use an example of `fromString` as well

Reply:
Already in the example? L16.